CMakeLists.txt

```diff
@@ -37,22 +57,82 @@ endif()
 # --------------------------------------------------------------------------------------------------------------------------------
 
 # define exported targets for nested project builds to consume
-add_library(Vulkan-Headers INTERFACE)
-target_include_directories(Vulkan-Headers INTERFACE "${CMAKE_CURRENT_SOURCE_DIR}/include")
-add_library(Vulkan::Headers ALIAS Vulkan-Headers)
-
-add_library(Vulkan-Registry INTERFACE)
-target_include_directories(Vulkan-Registry INTERFACE "${CMAKE_CURRENT_SOURCE_DIR}/registry")
-add_library(Vulkan::Registry ALIAS Vulkan-Registry)
-
-install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/include/vulkan" DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
-install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/registry" DESTINATION ${CMAKE_INSTALL_DATADIR}/vulkan)
-
-# uninstall target
-if(NOT TARGET uninstall)
-    configure_file("${CMAKE_CURRENT_SOURCE_DIR}/cmake/cmake_uninstall.cmake.in"
-                   "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
-                   IMMEDIATE
-                   @ONLY)
-    add_custom_target(uninstall COMMAND ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake)
+add_library(Headers INTERFACE)
```

Comment:
No need to rename the target (and breaks downstream projects). You can rely on https://cmake.org/cmake/help/git-stage/prop_tgt/EXPORT_NAME.html to export the target with another name.